src/main/java/com/iexec/core/replicate/ReplicatesService.java

```diff
@@ -404,26 +490,31 @@ private boolean verifyStatus(String chainTaskId, String walletAddress, Replicate
         }
     }
 
-    private boolean updateReplicateWeight(String chainTaskId, Replicate replicate) {
+    private boolean isWorkerWeightPresent(String chainTaskId,
+                                          Replicate replicate,
+                                          UpdateReplicateStatusArgs updateReplicateStatusArgs) {
```

Comment:
Since we're at it, isn't it enough to pass `workerWeight` as an argument to this method instead of `UpdateReplicateStatusArgs`?

Reply:
That could be a thing. But the current method name implies that we are looking for the `workerWeight` in an object. To be clearer, we should rephrase it to `isWorkerWeightCorrect` - or a better phrasing if you find one, I'm not 100% satisfied with this one.

Reply:
I generally tend to use `validate<target>` when it comes to checking variables or states. In this case, it would be `validateWorkerWeight()`. I also don't mind if you keep the method as it is, we can enhance it in coming iterations.

Reply:
Thanks, `validate<target>` seems better. That's fixed!